documentation/existing.md

```diff
@@ -186,66 +199,63 @@ Note: **```vm-name``` parameter is introduced in 1.6.4 release.** Both ```vm-uui
 
         cat /sys/class/dmi/id/product_serial | sed -e 's/^VMware-//' -e 's/-/ /' | awk '{ print toupper($1$2$3$4 "-" $5$6 "-" $7$8 "-" $9$10 "-" $11$12$13$14$15$16) }'
 
-* `datastore` is the default datastore used for provisioning volumes using storage classes. If datastore is located in storage folder or datastore is member of datastore cluster, make sure to specify full datastore path. Make sure vSphere Cloud Provider user has Read Privilege set on the datastore cluster or storage folder to be able to find datastore.
+* `datastore` is the default datastore used for provisioning volumes using storage classes. If datastore is located in storage folder or datastore is the member of datastore cluster, make sure to specify full datastore path. Make sure vSphere Cloud Provider user has Read Privilege set on the datastore cluster or storage folder to be able to find datastore.
   * For datastore located in the datastore cluster, specify datastore as mentioned below
-
+        ```
         datastore = "DatastoreCluster/datastore1"
-
+        ```
   * For datastore located in the storage folder, specify datastore as mentioned below
-
+        ```
         datastore = "DatastoreStorageFolder/datastore1"
+        ```
 
-#### Step 6: Add flags to controller-manager, API server and Kubelet
+## 3. Add flags to controller-manager, API server and Kubelet
 
-##### For Kubernetes version 1.9.x
+### For Kubernetes version 1.9.x
 
-* Add following flags to 
-  - kubelet running on master node 
-  - controller-manager manifest file
-  - API server manifest file
+* Add following flags to the kubelet configuration, controller-manager manifest file and API server manifest file on the master node.
 
-        --cloud-provider=vsphere
-        --cloud-config=<Path of the vsphere.conf file>
+    ```
+    --cloud-provider=vsphere
+    --cloud-config=<Path of the vsphere.conf file>
+    ```
 
 * Add following flags to kubelet running on each worker node.
 
-        --cloud-provider=vsphere
-
-##### For Kubernetes version 1.8.x or below
+    ```
+    --cloud-provider=vsphere
+    ```
 
-Add following flags to 
-- kubelet running on all nodes
-- controller-manager manifest file (typically running on master node)
-- API server manifest file (typically running on master node)
+### For Kubernetes version 1.8.x or below
 
-        --cloud-provider=vsphere
-        --cloud-config=<Path of the vsphere.conf file>
+* Add following flags to the kubelet configuration, controller-manager manifest file and API server manifest file on the master node.
 
-Manifest files for API server and controller-manager are generally located at `/etc/kubernetes`
+    ```
+    --cloud-provider=vsphere
+    --cloud-config=<Path of the vsphere.conf file>
+    ```
 
-#### Step 7: Restart Kubelet on all nodes
+## 4. Restart Kubelet on all nodes
 
 * Reload kubelet systemd unit file using ```systemctl daemon-reload```
 * Restart kubelet service using ```systemctl restart kubelet.service```
```

Comment:
Maybe worth mentioning here that this would restart the k8s control services too. Basically after updating k8s manifest files, we need to restart all k8s services.